po/zh-CN.po

```diff
@@ -2054,7 +2054,7 @@ msgstr ""
 
 #: src/welcome-day-1.md:36
 msgid "Morning: 9:00 to 12:00,"
-msgstr "早上：9:00 到 12:00，"
+msgstr "早上：9:00 到 12:00,"
```

Comment:
Why change to English comma?

Reply:
<img width="941" alt="image" src="https://github.com/google/comprehensive-rust/assets/11788119/19142645-ae21-4299-9446-e2703df40ca1">

If you use Poedit locally, you'll be prompted to make this change, or you'll be warned of an error!

Reply:
@SketchK, let's ignore the warning — if you find a way to turn it off, please help us document it.

Reply:
Please edit it back to use `，` and just ignore the warning for now.